ironfish-cli/src/commands/status.ts

```diff
@@ -8,6 +8,8 @@ import { Assert } from 'ironfish'
 import { IronfishCommand } from '../command'
 import { RemoteFlags } from '../flags'
 
+const BYTES_PER_MB = 1024 * 1024
```

Comment:
Don't use this use https://github.com/iron-fish/ironfish/blob/master/ironfish/src/utils/file.ts#L27

Reply:
Ah didn't realize this was a util. Updated.